### DISCUSSION TITLE
Integration to ocrmypdf

### DISCUSSION THREAD

**Discussion by savikko** [PaddlePaddle/PaddleOCR#12174] [Ideas & Features]
PaddleOCR seems to be very nice way to OCR documents.

There is project called ocrmypdf https://github.com/ocrmypdf/OCRmyPDF which has plugin system, where HOCR -compliant OCR engines can be integrated (it is using currenctly Tesseract as OCR engine).

It would be nice to use PaddleOCR as OCR engine on ocrmypdf.

**Comment by UserWangZz** [comment]
Any question for paddleocr?

**Comment by UserWangZz** [comment]
Hello, you are very welcome to use PaddleOCR as the OCR engine for your project. We will provide you with help in accessing OCRmyPDF. If you have any questions about PaddleOCR and access, please feel free to ask us.

**Reply by abclution** [reply]
Hello @UserWangZz I think what @savikko  is trying to ask, is there a way for PaddleOCR to output [HOCR - compliant](https://en.wikipedia.org/wiki/HOCR) recognition so it can be used as a plugin for OCRmyPDF? I that PaddleOCR has its own output format already. It would be interesting to also have HOCR output as well. :)

**Reply by aaronk6** [reply]
Looks like someone vibe-coded it: https://github.com/clefru/ocrmypdf-paddleocr Haven’t tried it yet, but it looks promising.

**Comment by ghost** [comment]
I would also like this because Tesseract OCR isn't very good at handwritten documents. Hope this idea isn't dead after 8 months.
